services/src/unit-test/java/uk/gov/moj/sdt/services/messaging/IndividualRequestMdbTest.java

```diff
@@ -153,11 +140,20 @@ public void readMessageDataAccessException() throws JMSException {
     public void readMessageInvalidObject() throws JMSException {
         individualRequestMdb = new IndividualRequestMdb(mockTargetSubmissionService);
 
-        final Message message = EasyMock.createMock(Message.class);
+        final Message message = mock(Message.class);
 
         individualRequestMdb.readMessage(message);
 
-        Assert.assertTrue("Test successfully completed", true);
+        assertTrue(true,TEST_SUCCESSFULLY_COMPLETED);
+    }
+
+    @Test
+    void setMockTargetSubmissionServiceTest(){
+        IndividualRequestMdb individualRequestMdbMock = mock(IndividualRequestMdb.class);
```

Comment:
Could use the predefined individualRequestMdb field here, but if you do you'll need to create a different mocked instance of the target submission service object to set it with.

Reply:
the predefined isn't a mock so can't call verified on it.  Have left this as is.

Reply:
Looking at this again, the test should be testing an actual IndividualRequestMdb object as the other tests in the class do rather than a mock of it.

```
individualRequestMdb = new IndividualRequestMdb(mockTargetSubmissionService);

TargetApplicationSubmissionService mockSubmissionService = mock(TargetApplicationSubmissionService.class);
individualRequestMdb.setTargetApplicationSubmissionService(mockSubmissionService);
```

The getAccessibleField() method can then be used to check that the field has been set correctly.